distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc

```diff
@@ -0,0 +1,38 @@
+:title: Manual Removal of Expired Records
```

Comment:
In other places, I've seen metacards referred to a "catalog entries". What is our canonical term for talking about a metacard in the catalog? Is it "records" or "entries"?

Reply:
I just looked at the `help` info of the removeall command. It uses "records"

Reply:
I'm on board with "records" We do need to improve standardization of the terms.